cmd/tailscaled/tailscaled.go

```diff
@@ -52,6 +59,8 @@ func main() {
 		defaultTunName = "tun"
 	}
 
+	cleanup := getopt.BoolLong("cleanup", 0, "clean up system state and exit")
+	subproc := getopt.BoolLong("subproc", 0, "run without supervision (will fail to restore system configuration on panic)")
```

Comment:
can we find a better flag name instead of `--subproc`? we already have `--debug` and `--fake` and now also `--cleanup`. They all sound kinda the same to me.

Reply:
I'm actually not happy about `--cleanup` in particular, since it's semantically more of an "action". Perhaps it is a good moment to introduce a positional argument for that? `subproc` could then be subsumed into that. Could be `tailscaled [flags ..] action` where `action` is in `cleanup, daemon, supervisor` with `supervisor` as the default? Then `--subproc` can also be replaced by some action value not displayed to the user (`supervisee`?).